portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getComment() {
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  const comments = document.getElementById("server");
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  // Get nested Json Objects
+  const info = fetch("/data")
+    .then((info) => info.json())
+    .then((info) => {
+      if (!("messageInfo" in info)) {
```

Comment:
Don't use `in` for JavaScript because it also checks the prototype chain. Prefer `info["messageInfo"] != null`.

Reply:
done